### DISCUSSION TITLE
Hours

### DISCUSSION THREAD

**Discussion by KatarzynaChmielecka** [DenverCoder1/github-readme-streak-stats#341] [Q&A]
On which hour (and time zone) streak is counted? Yesterday I committed 3 times and my streak now is 0.

**Comment by DenverCoder1** [comment]
I believe it is supposed to be midnight UTC time, but the GitHub API has frequently been inconsistent.

The only thing I recommend as a temporary solution is making commits have different date or time - https://github.com/DenverCoder1/github-readme-streak-stats/blob/main/docs/faq.md#why-doesnt-my-streak-stats-match-my-contribution-graph

It seems now that your streak shows 12 for me, so it may have been resolved automatically.

**Reply by KatarzynaChmielecka** [reply]
Yes, I forgot to write it here. But really I had 0 days earlier. So streak time is rather a lottery ... ;)

**Reply by KatarzynaChmielecka** [reply]
Hi, I have that problem again. Today I tried the second solution from your link. I put it in the terminal and pushed. git commit --date="2022-11-23 12:00" -m "Test commit" --allow-empty but on gh repo this commit isn't with the date I choose- it's ok?
